nservicebus/azure/azure-servicebus-transport.md

```diff
@@ -49,43 +49,44 @@ The default way to set the connection string is using the .NET provided `connect
 
 snippet:AzureServiceBusConnectionStringFromAppConfig
 
-For more detail see [Configuration Connection Strings](https://azure.microsoft.com/en-us/documentation/articles/service-bus-dotnet-how-to-use-topics-subscriptions/#set-up-a-service-bus-connection-string)
+For more details refer to [Configuration Connection Strings](https://azure.microsoft.com/en-us/documentation/articles/service-bus-dotnet-how-to-use-topics-subscriptions/#set-up-a-service-bus-connection-string) document.
 
 
-## Configuring in Detail
+## Detailed configuration
 
-If you need fine grained control on how the Azure Service Bus transport behaves, you can override the default settings by adding a configuration section called `AzureServiceBusQueueConfig` to your web.config or app.config files. For example:
+The default settings can be overriden by adding a configuration section called `AzureServiceBusQueueConfig` to the web.config or app.config files:
```

Comment:
Is this targeting core v5 or v6? In case of v6, this won't be relevant since we'll obsolete config sections.